modin/pandas/test/test_io.py

```diff
@@ -60,25 +62,9 @@
 TEST_GBQ_FILENAME = "test_gbq."
 SMALL_ROW_SIZE = 2000
 
-
-def eval_io(path, fn_name, comparator=df_equals, cast_to_str=False, *args, **kwargs):
-    def applyier(module, *args, **kwargs):
-        result = getattr(module, fn_name)(*args, **kwargs)
-        # There could be some missmatches in dtypes, so we're
-        # casting the whole frame to `str` before comparison.
-        # See issue #1931 for details.
-        if cast_to_str:
-            result = result.astype(str)
-        return result
-
-    eval_general(
-        pd,
-        pandas,
-        applyier,
-        path=path,
-        *args,
-        **kwargs,
-    )
+test_data_dir = os.path.join(os.path.dirname(__file__), "read_csv_data")
```

Comment:
Variable in global context should be in upper case.

Reply:
Corrected.